readme.txt

```diff
@@ -141,6 +141,10 @@ All is not lost!
 
 [Complete CHANGELOG.md maintained on Github](https://github.com/auth0/wp-auth0/blob/master/CHANGELOG.md)
 
+= 3.6.1 =
```

Comment:
what's the point of having this _incomplete_ changelog? is this for the plugin page?

Reply:
Yes ... people like to see something there. I didn't want to recreate the whole thing :)